source/extensions/load_balancing_policies/client_side_weighted_round_robin/client_side_weighted_round_robin_lb.h

```diff
@@ -82,6 +89,8 @@ class ClientSideWeightedRoundRobinLoadBalancer : public Upstream::ThreadAwareLoa
       return weight_;
     }
 
+    OrcaLoadReportHandlerSharedPtr report_handler_;
```

Comment:
Do we want this pointer Shared or Weak? We probably don't need to update weights if LB policy is deleted, right?

Reply:
Yeah. But shared pointer is harmless and may be better performance because we needn't to check it lifetime and create a temporary copy for every request.

I think initially we propose to use weak pointer at previous implementation is because the `OrcaLoadReportCallbacks` depends on the worker local lb in one of commit version. But After review the latest code, I found it's unecessary.